lib/tasks/backfill_in_person_pending_at.rake

```diff
@@ -0,0 +1,68 @@
+namespace :profiles do
+  desc 'Backfill the in_person_verification_pending_at value column.'
+
+  ##
+  # Usage:
+  #
+  # Print pending updates
+  # bundle exec rake profiles:backfill_in_person_verification_pending_at
+  #
+  # Commit updates
+  # bundle exec rake profiles:backfill_in_person_verification_pending_at UPDATE_PROFILES=true
+  #
+  task backfill_in_person_verification_pending_at: :environment do |_task, _args|
+    ActiveRecord::Base.connection.execute('SET statement_timeout = 60000')
+
+    update_profiles = ENV['UPDATE_PROFILES'] == 'true'
+
+    profiles = Profile.where(deactivation_reason: 'in_person_verification_pending')
```

Comment:
SInce we will be writing timestamps to profiles as of the deploy we want to ignore those profiles that already have the timestamp. `profiles = Profile.where(deactivation_reason: 'in_person_verification_pending', in_person_verification_pending_at: nil)

Reply:
Changed.